pyproject.toml

```diff
@@ -1,2 +1,58 @@
 [tool.black]
 target-version = ['py310']
+
+[tool.poetry]
+name = "screenpy"
+version = "4.0.2"
+description = "Screenplay pattern base for Python automated UI test suites."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+license = "MIT"
+#packages = []
+repository = "https://github.com/ScreenPyHQ/screenpy"
+documentation = "https://screenpy-docs.readthedocs.io"
```

Comment:
Does this stuff need to be here if it's also in the `metadata` stuff you added in `__version__.py`? It would be annoying if we have to remember to update it in two places.

Reply:
Unfortunately, yes.  It has to be in the toml for packaging reasons.   You'll notice all of the vars in `__version__` now get the info from `metadata` which originates from the toml now. 

I didn't feel confident enough in removing the `__version__` file.  I wasn't sure if it was required for something.

And `Eventually()` (*budum tiss*) we will want to remove the setup.py file.

Reply:
I will add a contributors section